python/cudf/cudf/core/multiindex.py

```diff
@@ -772,7 +787,7 @@ def _compute_validity_mask(self, index, row_tuple, max_length):
             ],
             axis=1,
         )
-        result = lookup.merge(data_table)["idx"]
+        result = postprocess(lookup.merge(data_table))["idx"]
```

Comment:
very minor nit: I think `postprocess` should be set just before it's used without the concat in between